labs/Lab_Data_Analytics/README.md

```diff
@@ -100,7 +100,7 @@ From the terminal, navigate to ```cd labs/Lab_Data_Analytics```
 
 ### Step 3.1 Navigate to the root directory of this lab
 
-Navigate to ```cd labs/Lab_Data_Analytics```
+Navigate to ```cd OpenAIWorkshop/labs/Lab_Data_Analytics```
```

Comment:
Step 1 has this step already to change directory. Could this line be confusing because it is a relative path and starts with "OpenAIWorkshop" ?

Reply:
Thank you. I updated the statement and pushed the changes.